<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordBook</class>
 <widget class="QWidget" name="WordBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>378</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 158, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>单词本</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>381</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{

}

QToolButton:hover{
	
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wordMinBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/min_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/min_hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wordCloseBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/close_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/close_hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{background-color: rgb(152, 214, 255);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="wordReviewBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>复习</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wordScanBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>浏览</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>370</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QToolButton" name="wordSynBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;同步&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;同步&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/refresh_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/refresh_hover.png);
}</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wordSettingBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/set_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/set_hover.png);
}</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(218, 218, 218);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QToolButton" name="wordAddBtn">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton{
	
	border-image: url(:/searchwindow/searchwindow/add_white.png);
}

QToolButton:hover{
	border-image: url(:/searchwindow/searchwindow/add_hover.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="wordEditBtn">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/edit_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/edit_hover.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="wordOmitBtn">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;删除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;删除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/close_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/close_hover.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="wordManageBtn">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;管理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;管理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton{
	border-image: url(:/searchwindow/searchwindow/open_normal.png);
}

QToolButton:hover{
	
	border-image: url(:/searchwindow/searchwindow/open_hover.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>365</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="font">
          <font>
           <family>SimSun-ExtB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <column>
          <property name="text">
           <string>单词</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>音标</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>解释</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>添加时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>完成度</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(180, 227, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>489</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>状态信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
